prisma/schema.prisma

```diff
@@ -222,6 +222,7 @@ model PlayerOwnedHouse {
   pool               Int?
   teleport           Int?
   garden_decoration  Int?
+  amulet_space       Int?
```

Comment:
no `_space` (none of the others have this in them)

Reply:
Removed `_space` and changed `AmuletSpace` to `Amulets` elsewhere.

This was tested by Lux, Pum and myself. I can get Pum to host on test server if needs be.